### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kisvegabor**
### Description of the feature or fix

A clear and concise description of what the bug or new feature is.

### Checkpoints
- [ ] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ ] Run `code-format.py` from the `scripts` folder. [astyle](http://astyle.sourceforge.net/install.html) needs to be installed.
- [ ] Update the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,140 +1,117 @@
...
-- [NXP MCUXpresso component](https://www.nxp.com/design/software/embedded-software/lvgl-open-source-graphics-library:LITTLEVGL-OPEN-SOURCE-GRAPHICS-LIBRARY)
-- [NuttX library](https://docs.lvgl.io/master/get-started/os/nuttx.html)
-- [RT-Thread RTOS](https://docs.lvgl.io/master/get-started/os/rt-thread.html)
-- CMSIS-Pack
+## Features
+
+### Free and Portable
+- **Fully portable** LVGL is a C (C++ compatible) library with no external dependencies. It can be compiled to any microcontrollers or microprocessors, with any (RT)OS to drive monochrome, ePaper, OLED or TFT displays, or even monitors.
+
+- **Free** LVGL comes with MIT licence you can easily use it in commercial projects too
```
Comment (embeddedt) [inline README.md]:
```suggestion
- **Free** LVGL is distributed under the MIT licence, so you can easily use it in commercial projects too.
```

File: README.md [hunk 2]
```diff
@@ -1,140 +1,117 @@
...
+
+- **Minimal requirements** C99 or newer compiler, minimum 32kB RAM and 128 kB flash, a frame buffer, and at least an 1/10 screen sized buffer for rendering. OS, External memory and GPU are supported but not required. 
+
+### Widgets, Styles, Layouts and more
+- **30+ built-in widgets**  Button, Label, Slider, Chart, Keyboard, Meter, Arc, Table and many more.
+
+- **Flexible style system**  ~100 style properties (e.g. radius, opacity, gradient, border, shadow, etc.) allows you to customize any part of the widgets (e.g. background, scrollbar, indicator, knob, etc) in any state (normal, pressed, check, focused, etc).
```
Comment (embeddedt) [inline README.md]:
```suggestion
- **Flexible style system**  ~100 style properties (e.g. radius, opacity, gradient, border, shadow, etc.) allow you to customize any part of the widgets (e.g. background, scrollbar, indicator, knob, etc) in any state (normal, pressed, check, focused, etc).
```

File: README.md [hunk 3]
```diff
@@ -1,140 +1,117 @@
...
+
+- **Various input devices** Supporting mouse, touchpad, Keypad, keyboard, external buttons, encoder. 
+
+- **Multiple display support** Use one MCU to drive more screens. 
+
+### Binding and build support
+
+- **Micropython Binding** Exposes [LVGL API in Micropython](https://blog.lvgl.io/2019-02-20/micropython-bindings)
+
+- **No custom build sysstem** LVGL is just a bunch of C files that can be easily integrated into any build system (built in support for Make and CMake)
```
Comment (embeddedt) [inline README.md]:
```suggestion
- **No custom build sysstem** LVGL is distributed as an organized collection of C files that can be easily integrated into any build system. Support for Make and CMake is included out of the box.
```

File: README.md [hunk 4]
```diff
@@ -1,140 +1,117 @@
...
+
+- **Develop on PC** Use the same UI code on PC and on the embedded hardware.
```
Comment (embeddedt) [inline README.md]:
```suggestion
- **Develop on PC** Use the same UI code on PC and embedded hardware. 
```

File: README.md [hunk 5]
```diff
@@ -1,140 +1,117 @@
...
-<a href="https://forum.lvgl.io">Forum</a>  &middot;
-<a href="https://lvgl.io/services">Services</a>  &middot;
-<a href="https://docs.lvgl.io/master/examples.html">Interactive examples</a>
-</h4>
+## Overview
 
+LVGL is the most popular free and open source embedded graphics library to create beautiful UIs for any MCU, MPU and display type. It's supported by industry leading vendors and projects like  **Arm, STM32, NXP, Espressif, Nuvoton, Arduino, RT-Thread, Zephyr, NuttX, Adafruit and many more**.
 
-**English** | [中文](./README_zh.md) | [Português do Brasil](./README_pt_BR.md)
+It has all the features to create modern and beautiful GUIs: 30+ built-in widgets, powerful style system, web inspired layout managers, and typography system supporting many languages. To integrate LVGL into your platform all you need is at least **32kB RAM and 128 kB flash**, a C compiler, a frame buffer, and at least an 1/10 screen sized buffer for rendering.
```
Comment (embeddedt) [inline README.md]:
```suggestion
It has all the features to create modern and beautiful GUIs: 30+ built-in widgets, a powerful style system, web inspired layout managers, and a typography system supporting many languages. To integrate LVGL into your platform, all you need is at least **32kB RAM and 128 kB flash**, a C compiler, a frame buffer, and at least an 1/10 screen sized buffer for rendering.
```

**Review by embeddedt — COMMENTED** [review]
I just have a few grammatical suggestions.

**Comment by kisvegabor** [comment]
Thank you very much! ^^

**Comment by kisvegabor** [comment]
@iamrameffort @mysterywolf 
We have updated https://lvgl.io and I aligned the first half of the README with the new communication directions. 

So could you update the translated README's?

**Comment by mysterywolf** [comment]
OK I will translate them in the next next week. I am busy during this period.

**Comment by kisvegabor** [comment]
Thank you Meco.

I merge this PR, feel free to open other one for the translation.

**Comment by iamrafaelmelo** [comment]
> @iamrameffort @mysterywolf We have updated https://lvgl.io and I aligned the first half of the README with the new communication directions.
> 
> So could you update the translated README's?

OK, I will translate later this week

**Comment by kisvegabor** [comment]
Thank you, Rafael.

**Pull request merged**